extensions/wikia/WikiFeatures/WikiFeaturesHelper.class.php

```diff
@@ -41,7 +40,6 @@ class WikiFeaturesHelper extends WikiaModel {
 	// no need to add feature to $release_date if not require "new" flag
 	public static $release_date = array (
 		'wgEnableChat' => '2011-08-01',
-		'wgEnableAchievementsExt' => '2012-02-12',
```

Comment:
https://wikia-inc.atlassian.net/browse/MAIN-10172 - ticket to upload i18n after this removal